specs/trip_dispatch_spec.rb

```diff
@@ -40,30 +41,30 @@
   end
 
 
-  # Uncomment for Wave 2
-  # describe "find_driver method" do
-  #   before do
-  #     @dispatcher = RideShare::TripDispatcher.new
-  #   end
-  #
-  #   it "throws an argument error for a bad ID" do
-  #     expect { @dispatcher.find_driver(0) }.must_raise ArgumentError
-  #   end
-  #
-  #   it "finds a driver instance" do
-  #     driver = @dispatcher.find_driver(2)
-  #     expect(driver).must_be_kind_of RideShare::Driver
-  #   end
-  # end
+  #Uncomment for Wave 2
+  describe "find_driver method" do
+    before do
+      @dispatcher = RideShare::TripDispatcher.new(USER_TEST_FILE,
+                                                 TRIP_TEST_FILE, DRIVER_TEST_FILE)
+    end
+
+    it "throws an argument error for a bad ID" do
+      expect { @dispatcher.find_driver(0) }.must_raise ArgumentError
```

Comment:
Minor comment: I think the correct behavior here would be for `TripDispatcher#find_driver` to return `nil` when no driver was found with the given ID.

The reasoning for this is that technically it's not an invalid argument -- the ID was actually an integer, it just didn't happen to match anything in our data set. Another reason for preferring to return `nil` is because Rails (which the `all` and `find` methods in these projects are based upon) also works that way.